src/AuditReader.php

```diff
@@ -189,12 +190,17 @@ public function clearEntityCache(): void
      * @throws NotAuditedException
      * @throws Exception
      * @throws ORMException
+     * @throws ORM2Exception
```

Comment:
```
Doctrine\ORM\Exception\ORMException
```
already exist in doctrine/orm 2.

And since `ORMException` extends `ORM2Exception` exception, I assume the doctrine codebase was updated and already throw `ORMException` everywhere ; so I would say we can remove all the `@throws ORM2Exception` tag, wdyt @franmomu ?

Reply:
Ah didn't check that, changed, thanks